data/human/weapons.txt

```diff
@@ -85,6 +85,28 @@ outfit "Quad Blaster Turret"
 		"hull damage" 6.6
 	description "A Quad Blaster Turret carries four guns on a single turret mount, doubling the firepower of an ordinary Blaster Turret for ships with enough space to install one and energy to drive it."
 
+outfit "Twin Blaster"
+	category "Guns"
+	cost 40000
+	thumbnail "outfit/twin blaster"
+	"mass" 12
+	"outfit space" -12
+	"weapon capacity" -12
+	"gun ports" -1
+	weapon
+		sprite "projectile/2x blaster"
+		sound "blaster"
+		"hit effect" "blaster impact"
+		"inaccuracy" 3
+		"velocity" 10.625
+		"lifetime" 48
+		"reload" 12
+		"firing energy" 11.6
+		"firing heat" 36
+		"shield damage" 21.2
+		"hull damage" 13.2
+	description "The Energy Blaster's ubiquity, low price, and stable performance have made it irresistible to tinkerers from the day it was first introduced. These blasters have been bolted together to fit within a single gun port, offering twice the damage of a single blaster at the cost of twice the heat and power consumption."
```

Comment:
```suggestion
	description "The Energy Blaster's ubiquity, low price, and stable performance have made it irresistible to tinkerers from the day it was first introduced. These blasters have been bolted together to fit within a single gun port, offering twice the damage of a single blaster at the cost of twice the heat and power consumption, plus a modicum of added weight.
```
Doesn't have to be this exactly, but if we're specifically calling out the exchange, I'd like to head off a million and one "bug reports" about the appropriate weight increase.